suite-common/wallet-core/src/token-definitions/tokenDefinitionsThunks.ts

```diff
@@ -0,0 +1,35 @@
+import { createThunk } from '@suite-common/redux-utils';
+import { Network } from '@suite-common/wallet-config';
+
+import { selectSpecificTokenDefinition } from './tokenDefinitionsSelectors';
+
+const actionsPrefix = '@common/wallet-core/token-definitions';
+
+export const getTokenDefinitionThunk = createThunk(
+    `${actionsPrefix}/getTokenDefinition`,
+    async (
+        params: { network: Network; contractAddress: string },
+        { getState, rejectWithValue },
+    ) => {
+        const { network, contractAddress } = params;
+        const { isTokenKnown } =
+            selectSpecificTokenDefinition(getState(), network.symbol, contractAddress) || {};
+
+        if (isTokenKnown === undefined) {
+            try {
+                const fetchedTokenDefinition = await fetch(
```

Comment:
are you sure that this will work? you are not returning these fetched values anywhere so you can't use them anywhere in `fullfiled` in reducer etc.

Reply:
It does not return any values, i just check that the link exist (HEAD request)